BinaryObjectScanner/FileType/RAR.cs

```diff
@@ -91,5 +103,71 @@ public bool Extract(Stream? stream, string file, string outDir, bool lookForHead
             return false;
 #endif
         }
+#if NET462_OR_GREATER || NETCOREAPP
+        /// <inheritdoc cref="IExtractable.Extract(Stream?, string, string, bool)"/>
+        // Extraction method for non-solid archives. This iterates over each entry in the archive to extract every file
+        // individually, in order to extract all valid files from the archive.
```

Comment:
These should go between `<summary>` tags. Please look for another method that uses those for more details.